detection_rules/schema.py

```diff
@@ -110,6 +128,7 @@ class SiemRuleApiSchema(jsl.Document):
 
     actions = jsl.ArrayField(required=False)
     author = jsl.ArrayField(jsl.StringField(default="Elastic"), required=True, min_items=1)
+    building_block_type = jsl.StringField(required=False)
```

Comment:
Defaults to `undefined`, but if specified during the rule creation flow we set to `default` ([source](https://github.com/elastic/kibana/blob/d5558cd4bf8dc8d1f6ba9b39267f7d832f9bc14a/x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/create/helpers.ts#L141)). Implemented as a `string` to support future building block types.